documentation/docs/25-build-and-deploy/50-adapter-static.md

```diff
@@ -162,8 +162,11 @@ jobs:
     environment:
       name: github-pages
       url: ${{ steps.deployment.outputs.page_url }}
-    
+
     steps:
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
```

Comment:
What does this do?

Reply:
looks like it sets the base path and origin:

https://github.com/actions/configure-pages/blob/f156874f8191504dae5b037505266ed5dda6c382/src/set-pages-config.js#L55

Reply:
so in theory the svelte-config part could be simplified to remove the check for dev mode? I'll try it out ...

Reply:
I haven't read this fully but looks like @Rich-Harris had some reservations against the action in https://github.com/sveltejs/kit/pull/5845

Reply:
After trying it out it on a fresh project definitely makes things slightly easier - no need to a) check for dev mode in `svelte.config.js` and b) use that to toggle between the base path being empty for dev or set to an env variable. So it saves having to make 2 changes to the `svelte.config.js` file.

AFAIK the base path is _always_ going to be set to the repository name for gh-pages, so all the auto setting does is change the mechanics of _how_ that is done by setting the config directly before the build instead of indirectly via an env variable.

I don't mind removing it but I think it makes it easier for a new person to use as it's a couple fewer changes to make to a new project. All I had to do was add the `.github/workflows/deploy.yml` file (which can be "copy-pasta" with no changes, set the project to use the static adapter & pre-render and it worked.

Reply:
> it saves having to make 2 changes to the svelte.config.js file

I think we could simplify that some without adding this action (which is two lines by itself)

Right now we have in the docs:
```
const dev = process.argv.includes('dev');

/** @type {import('@sveltejs/kit').Config} */
const config = {
	kit: {
		paths: {
			base: dev ? '' : process.env.BASE_PATH,
		}
  }
}
```

It could probably be simplified to just:
```
/** @type {import('@sveltejs/kit').Config} */
const config = {
	kit: {
		paths: process.env.BASE_PATH ?? ''
  }
}
```